src/main/java/uk/gov/hmcts/reform/blobrouter/controllers/EnvelopeController.java

```diff
@@ -69,9 +71,10 @@ private EnvelopeInfo toResponse(Envelope dbEnvelope, List<EnvelopeEvent> dbEvent
         );
     }
 
-    private Instant toLocalTimeZone(Instant instant) {
+    private String toLocalTimeZone(Instant instant) {
         if (instant != null) {
-            return instant.atZone(EUROPE_LONDON_ZONE_ID).toInstant();
+            DateTimeFormatter formatter = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm:ss");
```

Comment:
previously we had 'T' between date and time, to follow ISO standard https://en.wikipedia.org/wiki/ISO_8601

Reply:
Updated the format